bikeshed/boilerplate/wasm/abstract.include

```diff
@@ -2,6 +2,6 @@
 
 [ABSTRACT]
 Part of a collection of related documents:
-the <a href="https://www.w3.org/TR/wasm-core/">Core WebAssembly Specification</a>,
-the <a href="https://www.w3.org/TR/wasm-js-api/">WebAssembly JS Interface</a>,
-and the <a href="https://www.w3.org/TR/wasm-web-api/">WebAssembly Web API</a>.
+the <a href="https://www.w3.org/TR/wasm-core-1/">Core WebAssembly Specification</a>,
```

Comment:
Nooo don't link to the numbered specs...

Reply:
These should be the un-numbered one?

Reply:
I mean, yeah, ideally people should not ever look at the numbered specs, I think. (I am assuming this text is under your control, not part of some process-mandated boilerplate, since I haven't seen similar boilerplate before.)

Reply:
For those particular links going versionless makes sense, so I've changed them.
In general, this is totally me trying to fit the process :-)